chandra_models/xija/acisfp/acisfp_spec.json

```diff
@@ -447,15 +489,14 @@
             "name": "heatsink__1cbat"
         }
     ],
-    "datestart": "2019:101:00:03:10.816",
-    "datestop": "2021:169:23:53:50.816",
+    "datestart": "2020:154:00:01:26.816",
+    "datestop": "2022:022:23:51:42.816",
     "dt": 328.0,
     "evolve_method": 2,
     "gui_config": {
-        "filename": "/Users/jzuhone/Source/chandra_models/chandra_models/xija/acisfp/acisfp_spec.json",
+        "filename": "/Users/jzuhone/acisfp_spec_wsun.json",
```

Comment:
Is this directory and file name change intended?

Reply:
Hi @jskrist, this change is a no-op for scheduling and load review purposes as the only tool that uses this filename is `xija_gui_fit` which is used for model fitting